app/views/registrations/_form.html.erb

```diff
@@ -0,0 +1,21 @@
+<%= form_for(@registration ||= Registration.new, remote: true) do |f| %>
```

Comment:
The problem is that I call the form from a view that is not linked to any action of `registrations_controller`